elytron/src/main/java/org/wildfly/extension/elytron/AuthenticationContextAssociationProcessor.java

```diff
@@ -34,12 +44,57 @@
  */
 class AuthenticationContextAssociationProcessor implements DeploymentUnitProcessor {
 
+    private static final String[] FILES = {"wildfly-config.xml", "META-INF/wildfly-config.xml"};
```

Comment:
If a DUP is now handling the parsing of the configuration should we allow it to be located in WEB-INF?

Reply:
If we want 100% backward compat then WEB-INF/classes and WEB-INF/classes/META-INF should probably also be present.

Reply:
+1